src/docs/sphinx/basicExamples/multiphaseFlowWithWells/Example.rst

```diff
@@ -27,14 +27,14 @@ This example is based on the XML file located at
 
 .. code-block:: console
 
-  ../../../../../inputFiles/compositionalMultiphaseWell/benchmarks/Egg/deadOilEgg_base_iterative.xml
+  ../../../../../inputFiles/compositionalMultiphaseWell/benchmarks/Egg/deadOilEgg_benchmark.xml
```

Comment:
```suggestion
  inputFiles/compositionalMultiphaseWell/benchmarks/Egg/deadOilEgg_benchmark.xml
```
By convention, I think getting rid of the relative path operators would be better

Reply:
I agree but those relative paths are everywhere in the docs. @herve-gross could you please confirm what is the right way?